### ISSUE TITLE
Paladin's Retribution Aura is a mob magnet

### ISSUE THREAD

**Issue by Hr46ph** [dalaranwow/dalaran-wow#4577]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Once you have agro from a mob and run away from it it will chase you. With Retribution aura active, they won't give up or only after a VERY LONG time. Seriously, minutes. Sometimes they chase you the entire map. Switch to Devotion Aura, they give up within seconds.

**Expected behaviour**: Tell us what should happen instead
Mobs should not chase you forever and should give up similarly as with Devotion or no Aura active.

**Steps to reproduce the problem**:

1. Enable Devotion aura. Agro a mob and run the other way. They note the "entering combat" and "leaving combat messages". Do this a few times and time the average.
2. Switch to Retribution aura. Do the same, note the "entering combat", preferably with the same mob, running the same path. Note the time it takes for the mob to give up.
3. 

**Include proofs for this behaviour**
I would if I could. Not sure how I can prove this but every Retri in my guild confirms the behavior. No one bothered to file a bug report because its easy to avoid (switch auras).

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
This works already fine on PTR. The fix for it will be merged in the future on the live realm.
